ImageProcessing/src/image/geometric/Translate.md

```diff
@@ -0,0 +1,15 @@
+# image.geometric.translate
+
+The translate operator performs a geometric transformation which maps the position of each picture element in an input image into a new position in an output image. The image element located at (x1,y1) in the original is shifted to a new position (x2,y2) in the corresponding output image by displacing it through a user-specified translation (Ox,Oy).
+
```

Comment:
@Teymoor-Ali Can you give a real world example of a "user defined translation" used in image processing for translate?